assets/js/admin/form-builder.js

```diff
@@ -1347,6 +1350,53 @@
 				$( '.everest-forms-tab-content' ).scrollTop(0);
 				EVFPanelBuilder.switchToFieldOptionPanel( field_id );
 			} );
+		},
+
+		toggleLabelEdit( label, input ) {
+			$( label ).toggleClass( 'everest-forms-hidden' )
+			$( input ).toggleClass( 'everest-forms-hidden' )
+			
+			if ( $( input ).is(':visible') ) {
+				$( input ).focus()
+			}
+		},
+
+		bindToggleHandleActions: function () {
+			$( 'body' ).on( 'click', '.toggle-handle', function ( e ) {
+				let label = $( this ).data( 'label' )
```

Comment:
Please consistently use semicolons to terminate statements, you've terminated statements at some places, whilst left it out altogether in other places :)

Reply:
Okay. Thanks for the heads up.